app/src/routes/Games/containers/GamesContainer.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { getGames } from "../../../store/games";
+
+const propTypes = {
+    gamesList: PropTypes.array
+};
+
+const defaultProps = {
+    gamesList: []
+};
+
+const mapDispatchToProps = dispatch => ({
+    getGames: getGames(dispatch)
+});
+
+const mapStateToProps = (state) => {
+    if (state.games !== null) {
+        return {
+            gamesList: state.games.gamesList
+        };
+    } else {
+        return {
+            gamesList: []
+        };
+    }
+};
+
+const enhance = connect(mapStateToProps, mapDispatchToProps);
+
+class GamesContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = null;
+
+        this.getGames = this.getGames.bind(this);
+    }
+
+    getGames() {
+/*        this.props.getGames(({ ok }) => {
+            console.log("getGames w GamesContainer");
+            if (!ok) {
+                console.log("Getting list of games failed");
+            }
+        });*/
+    }
+
+    render() {
+        return (
+            <div className="gamesList">
+              {this.props.gamesList.map(game =>
```

Comment:
you can destructure `game` like this `.map(({ name, description, minPlayers, maxPlayers }) =>`